plotly/statistical_charts.py

```diff
@@ -0,0 +1,263 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "marimo",
+#     "numpy==2.2.5",
+#     "pandas==2.2.3",
+#     "plotly==6.0.1",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.13.6"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    import pandas as pd
+    import numpy as np
+    import plotly.express as px
+    import plotly.graph_objects as go
+    return go, mo, np, px
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+    # Statistical Charts
+
+    Welcome to this lesson on statistical charts with Plotly. In this lesson, you will learn to create and interact with four key chart types:
+
+    - **Box Plots:** Understand data spread, median, and outliers.  
+    - **Violin Plots:** Explore distribution shapes and density estimates.  
+    - **Histograms:** Analyze frequency distributions and experiment with different bin sizes.  
+    - **Error Bars:** Visualize measurement uncertainties and variability.
+
+    We will use the built-in tipstips dataset from Plotly for our examples. This dataset contains real-world information about restaurant bills, tips, and customer attributes, making it perfect for exploring statistical visualizations. Here are the first few rows of this dataset displayed below:
```

Comment:
```suggestion
    We will use the built-in [tips dataset](https://github.com/plotly/datasets/blob/master/tips.csv) from Plotly for our examples. This dataset contains real-world information about restaurant bills, tips, and customer attributes, making it perfect for exploring statistical visualizations. Here are the first few rows of this dataset displayed below:
```
Would hyperlink words to relevant sources (helps w/ accessibility).